src/test/java/net/logstash/logback/encoder/LogstashEncoderTest.java

```diff
@@ -107,43 +103,42 @@ public void basicsAreIncluded_logback12() throws Exception {
 
         when(logback11Support.isLogback11OrBefore()).thenReturn(true);
         final long timestamp = System.currentTimeMillis();
-        
+
         ILoggingEvent event = mockBasicILoggingEvent(Level.ERROR);
         when(event.getTimeStamp()).thenReturn(timestamp);
-        
+
         encoder.start();
         byte[] encoded = encoder.encode(event);
-        
+
         JsonNode node = MAPPER.readTree(encoded);
-        
+
         verifyBasics(timestamp, node);
     }
 
     protected void verifyBasics(final long timestamp, JsonNode node) {
-        assertThat(node.get("@timestamp").textValue()).isEqualTo(FastDateFormat.getInstance("yyyy-MM-dd'T'HH:mm:ss.SSSZZ").format(timestamp));
+        assertThat(node.get("@timestamp").textValue()).isEqualTo(String.valueOf(timestamp));
```

Comment:
I don't understand this change.  Can you clarify why the expected value is now a long value converted to a string, instead of a formatted timestamp string?

Reply:
I missunderstood the StringTimestampWriter, as a result I've done some wrong changes here